CPAC/utils/monitoring/custom_logging.py

```diff
@@ -2,8 +2,65 @@
 import logging
 import os
 
+from CPAC.utils.docs import docstring_parameter
+from CPAC.utils.monitoring.config import MOCK_LOGGERS
```

Comment:
If the `MOCK_LOGGERS` structure is empty, why do we define it as a separate object in a `config.py`, versus defining it here? Is this just in case there are future use-cases that may benefit from `MOCK_LOGGERS`? In those cases, does having the empty definition add anything? Forgive me if I'm totally missing the point of this piece — thanks!

Reply:
I don't think it would make a difference whether `MOCK_LOGGERS` is defined in place here or in a separate file, I just put it separate per [convention](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules). Besides having one fewer files, is there a reason to define in-place here? I don't really have a preference

Reply:
Not really — just that it would mean 1 fewer file in the tree, since the contents of it are just a single placeholder. I wondered if there was any voodoo I missed and didn't understand. No need to change :)